spec/index.html

```diff
@@ -429,11 +436,13 @@ <h2>Basic Concepts</h2>
     <p>
       The first goal of this specification is to allow a JSON-LD document to be
       processed and serialized into YAML, and then back into JSON-LD, without
-      losing any information.
+      losing any information.  
 
       This is always possible, because a YAML representation graph can always represent
```

Comment:
```suggestion
      losing any semantic information.</p>
    <p>
      This is always possible, because a YAML representation graph can always represent
```